packages/react-data-grid/src/common/editors/EditorContainer.tsx

```diff
@@ -117,7 +117,7 @@ export default class EditorContainer extends React.Component<Props, State> {
 
     return (
       <SimpleTextEditor
-        ref={this.editor as React.RefObject<SimpleTextEditor>}
+        ref={this.editor as unknown as React.RefObject<SimpleTextEditor>}
```

Comment:
![image](https://user-images.githubusercontent.com/567105/57929382-11823f00-78ab-11e9-8157-0cec6194abab.png)

Even though `input` is private... I wonder if this'll get fixed in the stable release.